css/properties/animation.json

```diff
@@ -106,6 +106,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "animation-timeline": {
+          "__compat": {
+            "description": "<code>animation-timeline</code> included in shorthand",
+            "support": {
+              "chrome": {
+                "version_added": "115",
+                "partial_implementation": true,
+                "notes": "Support is currently reset-only. Including <code>animation</code> resets a previously-declared <code>animation-timeline</code> value to <code>auto</code>, but <code>animation-timeline</code> cannot be set via <code>animation</code>."
```

Comment:
This came up in web-features, and I've filed https://crbug.com/337580010 to track this bug, and get clarity on whether it is a bug.